### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by julieschwartz18**
- Added a note to add ext- and api- FQDNs for XSOAR 8 OPP
- Added a link to the OPP guide for using certificates for authentication with EDLs in OPP

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EDL/Integrations/EDL/README.md [hunk 1]
```diff
@@ -184,7 +185,14 @@ To access the Export Indicators service by instance name, make sure ***Instance
  
 
 ### Access EDLs on Cortex XSOAR 8 and Cortex XSIAM
-
+EDLs running on tenants in Cortex XSOAR 8 require basic authentication. EDLs running on engines do not require basic authentication, but it is recommended.  
+For more information on using certificates for authentication with EDLs in Cortex XSOAR 8 OPP, see [HTTPS with a signed certificate](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/HTTPS-with-a-signed-certificate).  
+
+**Note:**  
+For Cortex XSOAR 8 OPP, you need to add the following DNS records:
```
Comment (richardbluestone) [inline Packs/EDL/Integrations/EDL/README.md:192]: For Cortex XSOAR On-prem....
We don't use OPP

File: Packs/EDL/Integrations/EDL/README.md [hunk 2]
```diff
@@ -184,7 +185,14 @@ To access the Export Indicators service by instance name, make sure ***Instance
  
 
 ### Access EDLs on Cortex XSOAR 8 and Cortex XSIAM
-
+EDLs running on tenants in Cortex XSOAR 8 require basic authentication. EDLs running on engines do not require basic authentication, but it is recommended.  
+For more information on using certificates for authentication with EDLs in Cortex XSOAR 8 OPP, see [HTTPS with a signed certificate](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/HTTPS-with-a-signed-certificate).
```
Comment (richardbluestone) [inline Packs/EDL/Integrations/EDL/README.md:189]: Cortex XSOAR On-prem

We don't use OPP

File: Packs/EDL/Integrations/EDL/README.md [hunk 3]
```diff
@@ -184,7 +185,14 @@ To access the Export Indicators service by instance name, make sure ***Instance
...
-
+EDLs running on tenants in Cortex XSOAR 8 require basic authentication. EDLs running on engines do not require basic authentication, but it is recommended.  
+For more information on using certificates for authentication with EDLs in Cortex XSOAR 8 OPP, see [HTTPS with a signed certificate](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/HTTPS-with-a-signed-certificate).  
+
+**Note:**  
+For Cortex XSOAR 8 OPP, you need to add the following DNS records:
+- ext-FQDN - The Cortex XSOAR DNS name mapped to the external IP address. For example, `ext-xsoar.mycompany.com`.
+- API-FQDN - The Cortex XSOAR DNS name mapped to the API IP address. For example, `api-xsoar.mycompany.com`.
+  
 For Cortex XSOAR 8 and Cortex XSIAM, you can only access the Export Indicators Service using a third-party tool such as cURL.
```
Comment (richardbluestone) [inline Packs/EDL/Integrations/EDL/README.md:196]: For Cortex XSOAR 8 Cloud...

**Review by richardbluestone — APPROVED** [review]

**Comment by richardbluestone** [comment]
We shouldn't use OPP - it should be On-prem

File: Packs/EDL/Integrations/EDL/README.md (hunk 1 above)
Comment (julieschwartz18) [reply Packs/EDL/Integrations/EDL/README.md]: Updated

File: Packs/EDL/Integrations/EDL/README.md (hunk 2 above)
Comment (julieschwartz18) [reply Packs/EDL/Integrations/EDL/README.md]: Updated

File: Packs/EDL/Integrations/EDL/README.md [hunk 4]
```diff
@@ -74,9 +74,9 @@ For Cortex XSIAM -
...
-| Certificate (Required for HTTPS)   | (For Cortex XSOAR 6.x) For use with HTTPS - the certificate that the service should use.  <br> (For Cortex XSOAR 8 and Cortex XSIAM) Custom certificates are not supported.                                                                                                                                                                                  | False        |
-| Private Key (Required for HTTPS)   | For Cortex XSOAR 6.x) For use with HTTPS - the private key that the service should use.  <br> (For Cortex XSOAR 8 and Cortex XSIAM) When using an engine, configure a private API key. Not supported on the Cortex XSOAR​​ or Cortex XSIAM server.                                                                                                                                                                                  | False        |
```
Comment (richardbluestone) [inline Packs/EDL/Integrations/EDL/README.md:79]: ...Cortex XSOAR or Cortex XSIAM tenant

File: Packs/EDL/Integrations/EDL/README.md [hunk 5]
```diff
@@ -172,29 +172,33 @@ In order to get the list of all available fields to search by, you can configure
...
 1. Navigate to **Settings > About > Troubleshooting**.
 2.  In the **Server Configuration** section, verify that the ***instance.execute.external*** key is set to *true*. If this key does not exist, click **+ Add Server Configuration** and add the *instance.execute.external* and set the value to *true*. See [this documentation](https://xsoar.pan.dev/docs/reference/articles/long-running-invoke) for further information.
 3. In a web browser, go to:
-    `https://*<xsoar_address>*/instance/execute/*<instance_name>*`
- 
-
-### Access EDLs on Cortex XSOAR 8 and Cortex XSIAM
-
-For Cortex XSOAR 8 and Cortex XSIAM, you can only access the Export Indicators Service using a third-party tool such as cURL.
-- On a tenant, use https://ext-<cortex-xsoar-address\>/xsoar/instance/execute/\<instance-name\>
```
Comment (richardbluestone) [inline Packs/EDL/Integrations/EDL/README.md:189]: Need to make sure that https://ext- is not clickable

File: Packs/EDL/Integrations/EDL/README.md (hunk 4 above)
Comment (julieschwartz18) [reply Packs/EDL/Integrations/EDL/README.md:79]: This part was removed. What was added refers to tenants

File: Packs/EDL/Integrations/EDL/README.md (hunk 5 above)
Comment (julieschwartz18) [reply Packs/EDL/Integrations/EDL/README.md:189]: This part was also removed and what was added was formatted so it's not clickable

**Pull request merged**